kie-server-parent/kie-server-common/src/test/java/org/kie/server/common/KeyStoreHelperUtilTest.java

```diff
@@ -16,32 +16,83 @@
 
 package org.kie.server.common;
 
+import java.net.URI;
+import java.nio.file.Paths;
+
+import org.drools.core.util.KeyStoreConstants;
+import org.drools.core.util.KeyStoreHelper;
+import org.junit.BeforeClass;
 import org.junit.Test;
 import org.kie.server.api.KieServerConstants;
 import org.kie.server.api.model.KieServerConfig;
 import org.kie.server.api.model.KieServerConfigItem;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
 import static org.kie.server.common.KeyStoreHelperUtil.loadControllerPassword;
 
 public class KeyStoreHelperUtilTest {
 
+    private static final String KEYSTORE_PATH = "target/keystore.jks";
+    private static final String KEYSTORE_PWD = "password";
+    private static final String KEYSTORE_ALIAS = "selfsigned";
+
+    @BeforeClass
+    public static void init() throws Exception {
+        // generate self signed certificate
+        String[] cmd = { "keytool", "-genkey",
+                "-keyalg", "RSA",
+                "-alias", KEYSTORE_ALIAS,
+                "-keystore", KEYSTORE_PATH,
+                "-storepass", KEYSTORE_PWD,
```

Comment:
I needed to add
```
                "-keypass", KEYSTORE_PWD,
```
otherwise the test was hanging for me infinitely.

Reply:
yeah after java 8 the keypass is mandatory. my bad for testing only in java 8